multicluster/hack/update-codegen-dockerized.sh

```diff
@@ -60,5 +59,8 @@ $GOPATH/bin/informer-gen \
   --output-package "${ANTREA_PKG}/multicluster/pkg/client/informers" \
   --go-header-file hack/boilerplate.go.txt
 
+go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.9.0
```

Comment:
I would suggest not to include unrelated changes in bug fix PRs, one PR should focus on one purpose, especially when it's a bug fix. Imageine how we handle it if this needs to backported. The side effect of bumping libraries would be backported or more efforts would be spent on dividing the patches.

Reply:
Other changes have been moved to this PR https://github.com/antrea-io/antrea/pull/3919